README.md
# page

# Getting Started with Create React App

This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).

## Available Scripts

In the project directory, you can run:

### `npm start`

Runs the app in the development mode.\
Open [http://localhost:3000](http://localhost:3000) to view it in your browser.

The page will reload when you make changes.\
You may also see any lint errors in the console.

### `npm test`

Launches the test runner in the interactive watch mode.\
See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.

### `npm run build`

Builds the app for production to the `build` folder.\
It correctly bundles React in production mode and optimizes the build for the best performance.

The build is minified and the filenames include the hashes.\
Your app is ready to be deployed!

See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.

### `npm run eject`

**Note: this is a one-way operation. Once you `eject`, you can't go back!**

If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.

Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.

You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.

## Learn More

You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).

To learn React, check out the [React documentation](https://reactjs.org/).

### Code Splitting

This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)

### Analyzing the Bundle Size

This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)

### Making a Progressive Web App

This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)

### Advanced Configuration

This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)

### Deployment

This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)

### `npm run build` fails to minify

This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)


changes.txt
<section className="hire-section">
        <h1>(What are you working on?)(Get In Touch)</h1>
        <p>let's have a conversation! I'd love to 
            hear about what you're working on and find
            a way to work together.
        </p>
        <a href="www.#.com"><button>Chat</button></a>
    </section>

<p>I'm a frontend web developer focused on building
            super fast and assessible human centered websites</p>



    {
        id: 3,
        coverPix: '',
        tittle: 'solo project',
        name: 'green earth',
        description: 'bussiness website for a consulting firm',
        lang: 'html, css & js',
        url: ''
    },

{
        id: 1,
        logo: 'html',
        name: 'html',
        description: 'hyper text markup language',
        level: '95%'
    },
    {
        id: 2,
        logo: 'css',
        name: 'css',
        description: 'casscading style sheet',
        level: '90%'
    },
    {
        id: 3,
        logo: 'js',
        name: 'js',
        description: 'javascript the web programming language',
        level: '60%'
    },
    {
        id: 4,
        logo: 'react',
        name: 'react',
        description: 'javascript library for dynamic frontend development',
        level: '80%'
    },
    {
        id: 5,
        logo: 'redux',
        name: 'redux',
        description: 'a state manager for react',
        level: '40%'
    },
    {
        id: 6,
        logo: 'responsive',
        name: 'responsive design',
        description: 'website design that fits perfectly on either web or mobile display',
        level: '95%'
    },
    {
        id: 7,
        logo: 'git',
        name: 'git & github',
        description: 'git repository hosting service',
        level: '90%'
    },
    {
        id: 8,
        logo: 'cmd',
        name: 'command line',
        description: 'codes & commands used while working with terminal',
        level: '85%'
    },
    {
        id: 9,
        logo: 'npm',
        name: 'npm',
        description: 'node package manager',
        level: '85%'
    },
    {
        id: 10,
        logo: 'api',
        name: 'restfull api',
        description: 'database storage for web resources',
        level: '60%'
    },
    {
        id: 11,
        logo: 'security',
        name: 'security',
        description: 'web security',
        level: '80%'
    },
    {
        id: 12,
        logo: 'production',
        name: 'production',
        description: 'website deployment',
        level: '90%'
    },
    {
        id: 13,
        logo: 'http',
        name: 'http',
        description: 'hypertext transfer protocol',
        level: '45%'
    },
    {
        id: 14,
        logo: 'json',
        name: 'json',
        description: 'language used to communicate with the server',
        level: '60%'
    },
    {
        id: 15,
        logo: 'node',
        name: 'node.js',
        description: 'backend js language',
        level: '40%'
    },
    {
        id: 16,
        logo: 'express',
        name: 'express.js',
        description: 'node feamework that provides broad features for web and mobile application',
        level: '29%'
    },
    {
        id: 17,
        logo: 'sql',
        name: 'postgreSQL',
        description: 'database coding and design',
        level: '50%'
    },

{
        id: 1,
        coverPix: 'flyo',
        tittle: 'solo project',
        name: 'fylo',
        description: 'This is a bussiness portfolio website template, that I created and it could be used for a tech company website',
        lang: 'html & css',
        url: 'https://arinzecyril.github.io/page/hi'
    },
    {
        id: 2,
        coverPix: 'foodmart',
        tittle: 'solo project',
        name: 'food mart',
        description: 'This is a simple website for a resturant or an eatery.',
        lang: 'html, css & js',
        url: 'https://arinzecyril.github.io/page/hi'
    },
    {
        id: 3,
        coverPix: 'chess',
        tittle: 'solo project',
        name: 'chess board',
        description: 'This is a fun projecti created. its basically a chess board with movable piece',
        lang: 'html, css & js',
        url: 'https://arinzecyril.github.io/page/hi'
    },
    {
        id: 4,
        coverPix: 'inputSearch',
        tittle: 'solo project',
        name: 'input event',
        description: 'This is a program that gets inputs and adds it to a list',
        lang: 'html, css & js',
        url: 'https://arinzecyril.github.io/page/hi'
    },
    {
        id: 5,
        coverPix: 'inputForm',
        tittle: 'solo project',
        name: 'fylo form',
        description: `This is an input form I created for the fylo wedsite`,
        lang: 'html, css & js',
        url: 'https://arinzecyril.github.io/page/hi'
    },



src/data/skillSet.js
export const SkillSet = [
    {
        id: 1,
        logo: 'html',
        name: 'html',
        description: '',
        level: '95%'
    },
    {
        id: 2,
        logo: 'css',
        name: 'css',
        description: '',
        level: '90%'
    },
    {
        id: 3,
        logo: 'js',
        name: 'js',
        description: '',
        level: '60%'
    },
    {
        id: 4,
        logo: 'react',
        name: 'react',
        description: '',
        level: '80%'
    },
    {
        id: 5,
        logo: 'redux',
        name: 'redux',
        description: '',
        level: '40%'
    },
    {
        id: 6,
        logo: 'responsive',
        name: 'responsive design',
        description: '',
        level: '95%'
    },
    {
        id: 7,
        logo: 'git',
        name: 'git & github',
        description: '',
        level: '90%'
    },
    {
        id: 8,
        logo: 'cmd',
        name: 'command line',
        description: '',
        level: '85%'
    },
    {
        id: 9,
        logo: 'npm',
        name: 'npm',
        description: '',
        level: '85%'
    },
    {
        id: 10,
        logo: 'api',
        name: 'restfull api',
        description: '',
        level: '60%'
    },
    {
        id: 11,
        logo: 'security',
        name: 'security',
        description: '',
        level: '80%'
    },
    {
        id: 12,
        logo: 'production',
        name: 'production',
        description: '',
        level: '90%'
    },
    {
        id: 13,
        logo: 'http',
        name: 'http',
        description: '',
        level: '45%'
    },
    {
        id: 14,
        logo: 'json',
        name: 'json',
        description: '',
        level: '60%'
    },
    {
        id: 15,
        logo: 'node',
        name: 'node.js',
        description: '',
        level: '40%'
    },
    {
        id: 16,
        logo: 'express',
        name: 'express.js',
        description: '',
        level: '29%'
    },
    {
        id: 17,
        logo: 'sql',
        name: 'postgreSQL',
        description: '',
        level: '50%'
    },
]

export const mainSkill = [
    {
        id: 1,
        logo: 'frontend',
        name: 'frontend development',
        describe: ``,
        level: '90%'
    },
    {
        id: 2,
        logo: 'backend',
        name: 'backend development',
        describe: '',
        level: '55%'
    },
    {
        id: 3,
        logo: 'database',
        name: 'database design',
        describe: '',
        level: '49%'
    },
]